README.pod

```diff
@@ -63,6 +63,27 @@ a subset of the functionality).
 It is based on Martin Berends' MiniDBI project, but unlike MiniDBI, DBDish
 aims to provide an interface that takes advantage of Perl 6 idioms
 
+=head2 Connecting to, and disconnecting from, a database
+
+For a database named I<hierarchy>, a connection may be established with any of
+the following syntaxes:
+
+    my ($dbh, $db);
+
+    $dbh = DBIish.connect("Pg", :database<hierarchy>);
+    $dbh.dispose;
+
+    $db = 'hierarchy';
+
+    $dbh = DBIish.connect("Pg", :database($db));
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", database => $db);
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", :$db);
```

Comment:
There is no PGUSER environment variable set.  I have never modified pg_hba.conf since installation.  I am not sure what you mean by the database "named as the _current_ user", but the code works even when I am not exploring any database via ```psql <database> <database_user>```.